Source/EasyNetQ/AdvancedBusExtensions.cs

```diff
@@ -1005,6 +1002,16 @@ public static void ExchangeDelete(this IAdvancedBus bus, Exchange exchange, bool
             .GetResult();
     }
 
+    /// <summary>
+    /// Delete an exchange
+    /// </summary>
+    /// <param name="bus">The bus instance</param>
+    /// <param name="exchange">The exchange to delete</param>
+    /// <param name="ifUnused">If set, the server will only delete the exchange if it has no queue bindings.</param>
+    /// <param name="cancellationToken">The cancellation token</param>
+    public static Task ExchangeDeleteAsync(this IAdvancedBus bus, Exchange exchange, bool ifUnused = false, CancellationToken cancellationToken = default)
```

Comment:
I do not see `public static Task QueueDeleteAsync(this IAdvancedBus bus, Queue queue, bool ifUnused = false, CancellationToken cancellationToken = default)` extension method.